pkg/pod/pod.go

```diff
@@ -724,6 +724,25 @@ func (builder *Builder) WithHostNetwork() *Builder {
 	return builder
 }
 
+// WithHostPid allows a pod to have access to the host process ID namespace.
+func (builder *Builder) WithHostPid() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Applying HostPID flag to pod's %s configuration", builder.Definition.Name)
```

Comment:
nit: if the pod name is "sun" - then this reads:
"Applying HostPID flag to pod's sun configuration"  which can be confusing.

Suggest to change:
"Applying HostPID flag to the configuration of pod %s"

Reply:
Please also include the namespace that the pod is running in